src/App.scss

```diff
@@ -1,38 +1,51 @@
 .container {
-  text-align: center;
-  width: 36rem;
-  max-width: 100%;
-  box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.2);
+  background-color: #fdf5e5;
 }
 
-.banner {
+.imageBanner {
+  width: 100%;
+  height: 100vh;
   display: flex;
   flex-direction: column;
-  // background-image: url("assets/arholma.jpeg");
-  background-repeat: no-repeat;
-  background-position: center top;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  background-image: url("ej.jpg");
   background-size: cover;
-  
-  // font-size: 4rem;
-  margin: 0;
-  padding: 1rem;
-  margin-block-start: 0;
+  background-position: center;
+  gap: 1rem;
+  img {
+    height: 100%;
+    object-fit: cover;
+    position: absolute;
+    top: 0;
+    z-index: -1;
+  }
```

Comment:
This makes it look well on mobile devices, but on desktop 16:9, only half of their heads are visible and on 32:9 only the sky and some treetops.